### PR TITLE
DUNE-JP cards

### PR DESCRIPTION
**Pull request by XGlitchy30**
Scripted the following DUNE-JP cards:
101201007    King Rex the Twin-Headed Dragon
101201008    King Rex the Dragon King
101201013    Infernoble Knight - Ricciardetto
101201014    Infernoble Knight - Turpin
101201017    Evoltile Phoris
101201018    Evolsaur Rios
101201040    Angelica, Princess of Noble Arms
101201043    Evolzar Razh
101201046    Grenosaurus the Magic-Head Cannon
101201048    Charlemagne
101201053    Jurassic Power
101201056    "Infernoble Arms - Almace"
101201057    The Noble Arms Museum
101201063    Transcendental Evolution Pill Beta
101201072    The Epic Poem of Charles

101201015    Nemurelia Réaliser, the Dreamaterializer Sleeping Princess
101201016    Nemurelia's Dreameater - Réveil
101201021    Ultimate Knight Alpha Ursatron
101201041    Ursarctic Polar Star
101201073    Nemurelia Löwe

Added the following setcodes:
!setname 0x507a Infernoble Knight
!setname 0x607a Infernoble Arms

### CODE REVIEW

File: scripts/DUNE-JP/c101201056.lua
Comment (Lyris12) [inline scripts/DUNE-JP/c101201056.lua:1]: bug: https://forum.duelistsunite.org/t/infernoble-arms-almace-first-effect-not-working/16170

**Review by Lyris12 — CHANGES_REQUESTED** [review]

File: scripts/DUNE-JP/c101201056.lua
Comment (XGlitchy30) [reply scripts/DUNE-JP/c101201056.lua:1]: I can't seem to reproduce the bug, I can activate the first effect with no issues. It doesn't seem the user attached a replay either to the post you linked, so I can't verify at all.

Comment (XGlitchy30) [reply scripts/DUNE-JP/c101201056.lua:1]: Just to be sure, did Omega update the setcode of the Infernoble Arms cards?

Comment (Lyris12) [reply scripts/DUNE-JP/c101201056.lua:1]: . That would explain it.

**Pull request merged**